cabot/cabotapp/views.py

```diff
@@ -562,6 +562,9 @@ class UserProfileUpdateAlert(LoginRequiredMixin, View):
     model = AlertPluginUserData
 
     def get(self, request, pk, alerttype):
+        if request.user.id != int(pk) and not request.user.is_staff:
```

Comment:
Would prefer only allowing superusers to view instead of staff (staff is anyone who can log into the admin console, superusers are the actual admins)